receiver/podmanreceiver/receiver_test.go

```diff
@@ -74,14 +68,14 @@ func TestScraperLoop(t *testing.T) {
 		}
 	}()
 
-	assert.NoError(t, r.Start(ctx, componenttest.NewNopHost()))
+	assert.NoError(t, r.start(ctx, componenttest.NewNopHost()))
+	defer func(){ assert.NoError(t, r.shutdown(ctx)) }()
```

Comment:
```suggestion
	defer func() { assert.NoError(t, r.shutdown(ctx)) }()
```
This is my fault, lint is complaining about this line 😞

Reply:
No issues, thanks for the heads up